ui/app/components/auth-form.js

```diff
@@ -244,9 +254,16 @@ export default Component.extend(DEFAULTS, {
   ),
 
   pollForOktaNumberChallenge: task(function* (nonce, mount) {
+    // yield for 1s to wait to see if there is a login error before polling
+    yield timeout(1000);
```

Comment:
while this is yield do we have some kind of loading state?

Reply:
I had to yield here because it would take them to a loading state if they typed in the wrong password and then take them to the error screen I created for the Okta Number Challenge instantly. The loading screen is used for waiting the user to click "Yes its me" on their Okta application. I talked to Ivana and she said if the username and password are wrong and the user doesn't proceed to MFA, then stay on the main sign-in screen and present the error.

I'm not sure if yielding is the best solution though because I noticed that sometimes if the API is slow the error thrown when the user types in a wrong password takes longer than 1s to be received, it would take them to the loading screen still. I was trying to look into waiting until the password authentication part is complete but I wasn't sure how that could be done as the polling for the Okta Number Challenge needs to be called right when the user clicks login.